localflavor/bg/validators.py

```diff
@@ -0,0 +1,68 @@
+from django.core.exceptions import ValidationError
+from django.utils.translation import ugettext_lazy as _
+
+from .utils import get_egn_birth_date
+
+
+def egn_validator(egn):
+    '''
+    Check Bulgarian unique citizenship number (EGN) for validity
+    More details https://en.wikipedia.org/wiki/Unique_citizenship_number
+    Full information in Bulgarian about algoritm is available here
+    http://www.grao.bg/esgraon.html#section2
+
+    '''
```

Comment:
Can you use `"""` for this comment and remove the space below? This is for all comments in this PR.